pkgs/tools/misc/shell-hist/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  name = "shell-hist";
```

Comment:
pname

Reply:
Oh `pname` as a key? I've not seen that before :open_mouth:

Reply:
yip the name gets then generated by pname + version.

Reply:
Cool, done! :slightly_smiling_face: